packages/react-ui/__tests__/widgets/BarWidgetUI.test.js

```diff
@@ -39,19 +42,21 @@ describe('BarWidgetUI', () => {
   };
 
   const WidgetMultiple = (props) => (
-    <BarWidgetUI
-      yAxisData={Y_AXIS_DATA_MULTIPLE}
-      xAxisData={X_AXIS_DATA}
-      series={SERIES}
-      labels={LABELS}
-      onSelectedBarsChange={() => {}}
-      {...props}
-    />
+    <IntlProvider locale='en'>
```

Comment:
@menusal  I think there is no need of wrapping every element on tests with its own IntProvider, as for all (or almost all) we will want 'en'. So for that my recommendation is to use the existing testUtils.js > `AllTheProviders` function, which automatically will wrap any 'render'-ed component with the corresponding providers